jsapp/js/actions.es6

```diff
@@ -537,6 +543,8 @@ actions.resources.createResource.listen(function(details){
   dataInterface.createResource(details)
     .done(function(asset){
       actions.resources.createResource.completed(asset);
+      var asset_type = asset.asset_type;
+      notify(t(`Successfully created ${asset_type}.`));
```

Comment:
Would this also notify the user when we create a form using formbuilder?
Is that something that we want?